engine/storage/datamotion/src/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -114,6 +142,17 @@ private boolean canHandle(DataStore dataStore) {
 
                     return true;
                 }
+            } else if (dataObject instanceof TemplateInfo) {
+                // If the storage system can clone volumes, we can cache templates on it.
+                String value = mapCapabilities.get(DataStoreCapabilities.CAN_CREATE_VOLUME_FROM_VOLUME.toString());
+                Boolean canCloneVolume = new Boolean(value);
```

Comment:
``new Boolean`` skips the constant pool -- placing unnecessary pressure on the garbage collector.  Use ``Boolean.valueOf`` instead.